doc/built_ins.rst

```diff
@@ -222,8 +222,18 @@ Adding new additional built-in operations
  5. Add meta-data describing this call to the appropriate file specified in
     the ``BUILTIN_DEFINITIONS_FILE`` in that source file. For dynamo0.3
     this is ``dynamo0p3_builtins_mod.f90``.
- 6. Document the new built-in in the documentation of the
-    relevant API (*e.g.* ``doc/dynamo0p3.rst``)
+ 6. Add relevant tests to the PSyclone test file for the API to be extended. 
```

Comment:
Please add "Modified I. Kavcic Met Office " at the appropriate place as you have done for other files.